src/engine.jl

```diff
@@ -40,6 +37,8 @@ type MSession
     end
 end
 
+MSession(options::String) = MSession(default_output_buffer_size, " " * strip(options))
```

Comment:
Documentation aside, it seems like a bad idea to have the two constructor behaves differently when there's no leading space in the option.